pkg/volume/iscsi/iscsi_util.go

```diff
@@ -153,9 +173,15 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) error {
 		return err
 	}
 
-	// check if the dev is using mpio and if so mount it via the dm-XX device
-	if mappedDevicePath := b.deviceUtil.FindMultipathDeviceForDevice(devicePath); mappedDevicePath != "" {
-		devicePath = mappedDevicePath
+	for _, path := range devicePaths {
+		if path == "" {
```

Comment:
It wont or very very rare chance, that check is added while considering the scenario where admin logged out from one path in between our fetch and use of that path.